packages/paste-website/src/components/CodeHighlight.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import {Box} from '@twilio-paste/box';
+
+export const CodeHighlight: React.FC<{children: React.ReactNode}> = ({children}) => (
```

Comment:
This already exists. It's called InlineCode https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/components/Typography.tsx#L29

Though, if you use markdown and it's back ticks, gatsby will mostly sub out the back ticks for `InlineCode`

```md
some text with `code` inline
```

Reply:
❓ are markdown tables preferred for docs pages? I used the table component here because I wanted to use a non-fixed layout. I noticed that it seems most tables are now markdown tables though.